src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +565,222 @@ subroutine DustEmission (bounds, &
          l = patch%landunit(p)
          g = patch%gridcell(p)
 
-         ! only perform the following calculations if lnd_frc_mbl is non-zero 
-
-         if (lnd_frc_mbl(p) > 0.0_r8) then
-
-            ! the following comes from subr. frc_thr_rgh_fct_get
-            ! purpose: compute factor by which surface roughness increases threshold
-            !          friction velocity (currently a constant)
-
-            frc_thr_rgh_fct = 1.0_r8
+         !--------------------------------------------------------------------------------------------------
+         ! put dust emission calculation here to output threshold friction velocity for the whole globe,
+         ! not just when lnd_frc_mbl = 0. Danny M. Leung 27 Nov 2021
+         bd = (1._r8-watsat(c,1))*2.7e3_r8      ![kg m-3] Bulk density of dry surface soil
```

Comment:
The magic numbers here should be turned into parameters: 2.7e3, 1.21, 100, 0.68

Reply:
These constants have been here for years. I can define them into parameters.